### DISCUSSION TITLE
Can nuclei scans be scheduled using crontab?

### DISCUSSION THREAD

**Discussion by PabloBree** [projectdiscovery/nuclei#865] [Q&A] [answered]
Hello everyone!

I just want to ask to see if its possible or not.. I'm made a python script to run nuclei, then i'm trying to schedule this python script so it can be executed every hour with crontab and realize that it doesn't work.. Saw that crontab executes all the code except the one for nuclei.. Any ideas if this is possible or maybe i'm missing a single step configuring cron?

**Comment by ehsandeep** [comment, marked as answer]
@PabloBree as cron is a system-level utility, nuclei should not affect it, make sure to debug the execution to see what actually causing the issue.
